ngx-tools/src/set-form-control-value/set-form-control-value.ts

```diff
@@ -0,0 +1,17 @@
+import { FormGroup } from '@angular/forms';
+
+
+export function setFormControlValue(form: FormGroup, controlName: string, controlValue: string): any {
```

Comment:
`controlValue` probably needs to be `any` in case the consumer ever needs to set the control to `null`.

Reply:
Other than `null`, would it be any other case? If not, should it be `string | null`?

Reply:
I would do `any` since that is how the typing is defined in the `setValue` api. [[reference]](https://angular.io/api/forms/FormControl#setValue)

I guess technically it could be anything.. a form control can have strings, numbers, null, undefined etc